### PR TITLE
Validate analog files

### PR DESCRIPTION
**Pull request by Twinters007**

### CODE REVIEW

File: wpilib/wpilib/analoginput.py [hunk 1]
```diff
@@ -326,10 +326,16 @@ def pidGet(self):
     def getSmartDashboardType(self):
         return "Analog Input"
 
+    def initTable(self, subtable):
+        if subtable is not None:
+            self._valueEntry = subtable.getValue("Value")
```
Comment (virtuald) [inline wpilib/wpilib/analoginput.py]: getValue != getEntry... I think getValue is equivalent to ntproperty.

Comment (Twinters007) [reply wpilib/wpilib/analoginput.py]: Ugh how many times did I do that...

**Comment by virtuald** [comment]
I'm not going to merge this until the tests pass. Which... can't really happen until ntcore is updated.

**Comment by Twinters007** [comment]
Agreed

File: wpilib/wpilib/analogaccelerometer.py [hunk 2]
```diff
@@ -44,6 +45,7 @@ def __init__(self, channel):
         LiveWindow.addSensorChannel("Accelerometer",
                                     self.analogChannel.getChannel(), self)
 
+
```
Comment (auscompgeek) [inline wpilib/wpilib/analogaccelerometer.py]: What's with this extra blank line?

File: wpilib/wpilib/analogaccelerometer.py [hunk 3]
```diff
@@ -12,6 +12,7 @@
 from .interfaces import PIDSource
 from .livewindow import LiveWindow
 from .livewindowsendable import LiveWindowSendable
+from networktables.networktable import NetworkTable
```
Comment (auscompgeek) [inline wpilib/wpilib/analogaccelerometer.py]: Just `from networktables import NetworkTable` should be fine. But this is fine too I guess.

File: wpilib/wpilib/analoginput.py [hunk 4]
```diff
@@ -47,7 +47,7 @@ def __init__(self, channel):
         :param channel: The channel number to represent. 0-3 are on-board 4-7 are on the MXP port.
         """
         
-        SensorBase.checkAnalogInputChannel(channel)
```
Comment (virtuald) [inline wpilib/wpilib/analoginput.py]: Why did you change this? Did SensorBase change? We have it as a static method.

Comment (Twinters007) [reply wpilib/wpilib/analoginput.py]: Looks like I dorked on how java works and thought it was part of this class. I’ll revert. ¯\_(ツ)_/¯

File: wpilib/wpilib/analogoutput.py [hunk 5]
```diff
@@ -28,7 +28,7 @@ def __init__(self, channel):
 
         :param channel: The channel number to represent.
         """
-        SensorBase.checkAnalogOutputChannel(channel)
```
Comment (virtuald) [inline wpilib/wpilib/analogoutput.py]: One more of these.
